woocommerce/wc-admin
# Packages – Components: Move proptype-validator utility into src

Only `src` files are transpiled for external use, so requiring this file from lib is breaking my jest tests. See https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/174

I've also bumped the version number, since I'll do a package release once this is merged (to fix my other PR).

### Detailed test instructions:

- Apply this PR, and run `npm run build:packages`
- `cd packages/components` and run `npm link`
- Switch over to `woocommerce-gutenberg-products-block` at PR 174, run `npm link @woocommerce/components`
- run the tests in the blocks plugin
- Expect: passing tests, no SyntaxError.
